src/DynamoRevit/DynamoRevit.csproj

```diff
@@ -160,4 +95,17 @@
   <ItemGroup>
     <Compile Remove="RevitSettings.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="DynamoVisualProgramming.Core" Version="3.0.0.7190" ExcludeAssets="runtime" GeneratePathProperty="true" />
+    <PackageReference Include="DynamoVisualProgramming.WpfUILibrary" Version="3.0.0.7190" ExcludeAssets="runtime" />
+    <PackageReference Include="Greg" Version="3.0.0.3175" ExcludeAssets="runtime" />
+    <PackageReference Include="GregRevitAuth" Version="2.0.8718.22924" />
+    <PackageReference Include="pythonnet_py38_win" Version="2.5.1" ExcludeAssets="runtime" />
+  </ItemGroup>
+  <Target Name="Copy dll" AfterTargets="Build">
+    <ItemGroup>
+        <DetectiveDll Include="$(PkgDynamoVisualProgramming_Core)\lib\net8.0\DynamoInstallDetective.dll" />
```

Comment:
Why doesn't this automatically get copied to the output path using `EnableDynamicLoading`?